_posts/2018-05-08-CocoaPods-1.6.0-beta.markdown

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title:  "CocoaPods 1.6.0 Beta Begins!"
+author: dnkoutso
+categories: cocoapods releases
+---
+
+_CocoaPods 1.6.0_ release will focus on stability, performance and scalability for large projects through a longer beta cycle.
+
+<!-- more -->
+
+We are happy to announce that the first beta of 1.6.0 is out! For this release, we are revamping the build settings generation process to be faster and easier to maintain going forward and we've enhanced integration of test specs!
+
+### Revamped Build Settings
+
+The generation of build settings (which reside within the `.xcconfig` files CocoaPods generates) has been completely re-written. The primary goal for this re-write was to clean up the code and make it scale much better for larger projects. 
+
+When contributors to CocoaPods make build setting changes it would often cause a cascading amount of performance regressions and trigger an unusual combination of edge cases. This made it very hard to test these changes and be certain that each fix works correctly for everyone.
+
+From a user point of view, this re-write should have zero effects when you compile your code. If there is anything that does not look right please file an [issue](https://github.com/CocoaPods/CocoaPods/issues/new) with us providing steps and a sample app showing the problem.
```

Comment:
`please file an issue, providing steps & a sample app that demonstrates the problem`